docs/readme.md

```diff
@@ -0,0 +1,25 @@
+# Taxonomy
+
```

Comment:
We may add a minimal intro here describing that we are generating two more trees out of the original one, picking nodes that meet some criteria (unclassified in one case, environmental samples in the other) and "extracting" all its ancestors and all its descendants.
I will add this after the other comments are resolved.